### DISCUSSION TITLE
Chinese garbled code

### DISCUSSION THREAD

**Discussion by TsinlingLi** [datapane/datapane#140] [Help] [answered]
Create a simple DF and generate Chinese garbled code after the report

df= pd.DataFrame([[1, '哈哈'], [2, '嘿嘿']], columns=['顺序', 'code'])

report = dp.Report(
  dp.Table(df, caption="试试", label='这就是试试label')
)
![image](https://user-images.githubusercontent.com/42768590/128193525-1249550e-371c-4771-9415-d22586e40af1.png)

Who knows how to solve this problem

**Comment by mands** [comment, marked as answer]
Thanks for raising and for the sample code - we'll add to our backlog and fix shortly.

**Reply by mands** [reply]
Could you provide some information on your system, e.g. OS, system locale, Python version, Pandas version, etc.?

**Reply by TsinlingLi** [reply]
Thank you for your reply
systeminfo
OS 名称:          Microsoft Windows 10 专业版
OS 版本:          10.0.19042 暂缺 Build 19042
OS 制造商:        Microsoft Corporation
OS 配置:          独立工作站
OS 构建类型:      Multiprocessor Free
注册的所有人:     China
注册的组织:       Home
产品 ID:          00330-80000-00000-AA339
初始安装日期:     2021-02-17, 18:51:46
系统启动时间:     2021-08-05, 19:44:56
系统制造商:       Micro-Star International Co., Ltd
系统型号:         MS-7C02
系统类型:         x64-based PC
处理器:           安装了 1 个处理器。
                  [01]: AMD64 Family 23 Model 113 Stepping 0 AuthenticAMD ~3600 Mhz
BIOS 版本:        American Megatrends Inc. 1.C0, 2019-09-24
Windows 目录:     C:\WINDOWS
系统目录:         C:\WINDOWS\system32
启动设备:         \Device\HarddiskVolume1
系统区域设置:     zh-cn;中文(中国)
输入法区域设置:   zh-cn;中文(中国)
时区:             (UTC+08:00) 北京，重庆，香港特别行政区，乌鲁木齐


Python 3.8.10

Name: numpy
Version: 1.21.0
Summary: NumPy is the fundamental package for array computing with Python.
Home-page: https://www.numpy.org
Author: Travis E. Oliphant et al.
Author-email: None
License: BSD
Location: c:\anaconda3\envs\env38\lib\site-packages
Requires:
Required-by: xgboost, scipy, scikit-learn, python-louvain, pandas, networkx, matplotlib, communities, bokeh

Name: pandas
Version: 1.3.0
Summary: Powerful data structures for data analysis, time series, and statistics
Home-page: https://pandas.pydata.org
Author: The Pandas Development Team
Author-email: pandas-dev@python.org
License: BSD-3-Clause
Location: c:\anaconda3\envs\env38\lib\site-packages
Requires: numpy, pytz, python-dateutil
Required-by: networkx

**Reply by mands** [reply]
Hi @TsinlingLi, thanks for the info! We've just released a fix for this that will be published to pip as 0.12.0 in a few days. Cheers!
